README.md

```diff
@@ -1 +1,9 @@
-# training-txome-prep
\ No newline at end of file
+# Salmon transcriptome index preparation
+
+This repository is intended to document how Salmon transcriptome indices are generated for RNA-seq and scRNA-seq training modules.
+Currently, this corresponds to our 2019 Houston workshop.
+
+We run all scripts in the Docker container for training (introduced here: [AlexsLemonade/RNA-Seq-Exercises](https://github.com/AlexsLemonade/RNA-Seq-Exercises/pull/22)).
+_Note: `optparse` is not installed on that Docker image but is required to generate the tx2gene files._
```

Comment:
Just to clarify, does this mean if you are to run this in the container, you need to install optparse before you run it?

Reply:
If so, should you just add a line in `scripts/get_tx2gene.R` that installs optparse if it is not installed?

Reply:
You are correct. This sounds like a good idea.